crates/task-impls/src/quorum_proposal.rs

```diff
@@ -492,7 +495,7 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>> QuorumProposalTaskState<TYPE
                 .await;
```

Comment:
Orthogonal to the previous discussion, but could we rename `view` to something like `proposal_view` or `previous_proposal_view`? I think `view` is a somewhat vague name, especially when there are 2-3 potential views floating around.

Also, should `self.latest_proposed_view` be updated somewhere here (after validation? not really sure, just wondering)?

Reply:
It's updated once the proposal task completes and the event is emitted.

Reply:
I'll adjust naming in the integration branch since this code isn't touch in this PR, but it is touched there. Sorry, there's a lot of changes in flight.